src/Aspire.Hosting/ProjectResourceBuilderExtensions.cs

```diff
@@ -207,43 +208,84 @@ private static IResourceBuilder<ProjectResource> WithProjectDefaults(this IResou
                 // If we have any endpoints & the forwarded headers wasn't disabled then add it
                 if (projectResource.GetEndpoints().Any() && !projectResource.Annotations.OfType<DisableForwardedHeadersAnnotation>().Any())
                 {
-                    context.EnvironmentVariables[AspNetCoreForwaredHeadersEnabledVariableName] = "true";
+                    context.EnvironmentVariables[AspNetCoreForwardedHeadersEnabledVariableName] = "true";
                 }
             });
         }
 
         if (excludeLaunchProfile)
         {
             builder.WithAnnotation(new ExcludeLaunchProfileAnnotation());
-            return builder;
```

Comment:
@davidfowl so if launch profile is excluded, we ignore any passed in `launchProfileName` (both before and after this change). It's slightly quirky, and throwing might be more correct, but that's a minor issue.